pkg/operator/v1helpers/helpers.go

```diff
@@ -259,3 +261,18 @@ func (agg aggregate) Error() string {
 func (agg aggregate) Errors() []error {
 	return []error(agg)
 }
+
+func MapToEnvVars(proxyConfig map[string]string) []corev1.EnvVar {
+	if proxyConfig == nil {
+		return nil
```

Comment:
i would return empty list maybe so we can append?

Reply:
append works with nil too.